gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/SnappyUtils.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2017 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+package com.pivotal.gemfirexd.internal.engine.ui;
+
+import java.util.Locale;
+
+public class SnappyUtils {
+
+  public enum StorageSizeUnits {
+    B, KB, MB, GB, TB, ANY
```

Comment:
Should add support for PB also. If we scale in some cases this should not be a re-work.

Reply:
Added support for PB (peta bytes).